target_stitch/__init__.py

```diff
@@ -462,6 +469,13 @@ def consume(self, reader):
             self.handle_line(line)
         self.flush()
 
+    def print_counts(self):
+        printstr = '\n----------------------'
+        for stream_id, stream_count in self.counts.items():
+            printstr += '\n{}: {}'.format(stream_id, stream_count)
+        printstr += '\n----------------------'
+        LOGGER.info(printstr)
```

Comment:
I think a LOGGER.info for each line here might be better than one log line with newline characters. That's what Stripe does https://github.com/singer-io/tap-stripe/blob/master/tap_stripe/__init__.py#L120-L126

Reply:
That's what I originally had, but during my first two test runs some other log lines worked their way into the summary since orca is logging at the same time as the target :(
```
2019-04-03 22:48:19,486Z target - ----------------------
2019-04-03 22:48:19,486Z   main - INFO State update: changing bookmarks.conversation_message_activity from "2019-04-03T22:45:01.805413+00:00" to "2019-04-03T22:48:19.395468+00:00"
2019-04-03 22:48:19,493Z   main - INFO State update: changing bookmarks.message_activity from "2019-04-03T22:45:01.805316+00:00" to "2019-04-03T22:48:19.395373+00:00"
2019-04-03 22:48:19,487Z target - messages: 2067
2019-04-03 22:48:19,487Z target - ----------------------
```

Reply:
The first `\n` is there to avoid misalignment.  WIthout it you get:
```
2019-04-03 22:48:19,486Z target - INFO ----------------------
2019-04-03 22:48:19,487Z target - messages: 2067
2019-04-03 22:48:19,487Z target - ----------------------
```
It's not a huge deal either way-- I can take it out.

Reply:
Ahhh, I see. No, that seems fine. :+1: